is_core/generic_views/table_views.py

```diff
@@ -135,31 +137,35 @@ def _get_header_order_by(self, model, full_field_name):
     def _get_field_labels(self):
         return self.field_labels
 
-    def _get_header_label(self, model, full_field_name, field_name):
+    def _get_field_or_method_label(self, model, field_name):
+        try:
+            field = model._meta.get_field(field_name)
+            if field.auto_created and (field.one_to_many or field.many_to_many):
+                return (
+                    getattr(field.field, 'reverse_verbose_name', None) or
+                    field.related_model._meta.verbose_name_plural
+                )
+            elif field.auto_created and field.one_to_one:
```

Comment:
Tady se ti opakuje ta podmínka `field.auto_created`, nechceš to zavřít pod jeden if?

Reply:
to bych to zas zbytečně zanořil